Marlin/src/lcd/dogm/marlinui_DOGM.h

```diff
@@ -124,6 +122,18 @@
     #define U8G_CLASS U8GLIB_SSD1306_128X64                     // 8 stripes
   #endif
 
+#elif ENABLED(FYSETC_242_OLED_12864)
+
+  // FYSETC OLED 2.42" 128 × 64 FULL GRAPHICS CONTROLLER
+
+  #define FORCE_SOFT_SPI                                        // SW-SPI
+
+  #if ENABLED(ALTERNATIVE_LCD)
+    #define U8G_CLASS U8GLIB_SSD1306_128X64_2X                  // 4 stripes
+  #else
+    #define U8G_CLASS U8GLIB_SSD1309_128X64_HAL
+  #endif
+
```

Comment:
@GerogeFu — Is this display sometimes SSD1306 and sometimes SSD1309, or is it always SSD1309 ?